packages/jest/src/plugins/plugin.spec.ts

```diff
@@ -81,6 +81,7 @@ describe('@nx/jest/plugin', () => {
                       },
                     ],
                     "metadata": {
+                      "atomizedTarget": undefined,
```

Comment:
Please get rid of the `undefined`. Only set the property when `options.ciTargetName` is defined. I don't really know why but `jest` sometimes isn't happy with this on different systems :shrug: